pkg/controller/hostpathprovisioner/contoller_test.go

```diff
@@ -191,7 +195,7 @@ var _ = Describe("Controller reconcile loop", func() {
 			},
 		}
 		crbNN := types.NamespacedName{
-			Name: "test-name",
+			Name: "hostpath-provisioner",
```

Comment:
Use the const you just create in common.go?

Reply:
Done